app/validators/claim/base_claim_validator.rb

```diff
@@ -358,10 +358,10 @@ def validate_too_far_in_past(start_attribute)
   end
 
   def has_higher_rate_mileage?
-    @record.expenses.map { |s| s.mileage_rate_id.eql?(2) }.any?
+    @record.expenses.where(mileage_rate_id: 2).any?
   end
 
   def increased_travel?
-    @record.expenses.map { |x| x.calculated_distance && (x.distance > x.calculated_distance) }.any?
+    @record.expenses.where('calculated_distance IS NOT NULL AND (distance > calculated_distance)').any?
```

Comment:
For SQL you might be able (he says over-confidently)  to get away with
```
where('distance > calculated_distance').any?
```
...as any expression with values evaluating to null in should return null, and therefore false

but either way 👍

Reply:
Good call, rebuilding, then I'll merge